lib/src/components/NxTextLink/__tests__/NxTextLink.test.tsx

```diff
@@ -5,114 +5,106 @@
  * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
  */
 import React from 'react';
-import { getShallowComponent } from '../../../__testutils__/enzymeUtils';
+import { rtlRender, rtlRenderElement } from '../../../__testutils__/rtlUtils';
 
 import NxTextLink from '../NxTextLink';
-import NxFontAwesomeIcon from '../../NxFontAwesomeIcon/NxFontAwesomeIcon';
-import { IconProp } from '@fortawesome/fontawesome-svg-core';
 
 describe('NxTextLink', function() {
-  const getShallow = getShallowComponent(NxTextLink, {});
+  const quickRender = rtlRender(NxTextLink, {});
+  const renderEl = rtlRenderElement(NxTextLink, {});
 
   it('renders an <a> with the nx-text-link class', function() {
```

Comment:
The class name is an implementation detail as far as RTL is concerned

Reply:
> The class name is an implementation detail as far as RTL is concerned

Removed here: 4e7e494279e16de3a908ade9be773ce6d2bd9065

Reply:
> Removed here: https://github.com/sonatype/sonatype-react-shared-components/commit/4e7e494279e16de3a908ade9be773ce6d2bd9065

IMO it was still worthwhile to test that it was an `<a>`, since `<a>` tags have special accessibility behaviors in the browser such as showing the hyperlink target in the bottom of the viewport when you hover the link.

Reply:
> > Removed here: [4e7e494](https://github.com/sonatype/sonatype-react-shared-components/commit/4e7e494279e16de3a908ade9be773ce6d2bd9065)
> 
> IMO it was still worthwhile to test that it was an `<a>`, since `<a>` tags have special accessibility behaviors in the browser such as showing the hyperlink target in the bottom of the viewport when you hover the link.

Added that part back: a5cc4005f6e6e5133b043cff46e94438a409293e